www/store/features/theme/themeSlice.js

```diff
@@ -22,8 +22,12 @@ export const pickTheme = ( theme ) => {
   return async function themePick( dispatch, getState ) {
     const state = getState();
     const user_id = state.user._id;
+    const currentTheme = window.localStorage.getItem('theme');
 
     window.localStorage.setItem( 'theme', theme );
+    document.querySelector('html').classList.remove( currentTheme );
```

Comment:
It would be better to set the dom node to a variable so that you don't need to query the dom twice:

```js
const html = document.querySelector('html');

html.classList.remove( currentTheme );
html.classList.add( theme );
```